--- a/Sources/CgRPC/src/core/ext/filters/client_channel/lb_policy/xds/xds.cc
+++ b/Sources/CgRPC/src/core/ext/filters/client_channel/lb_policy/xds/xds.cc
@@ -68,7 +68,7 @@
 #include <grpc/support/string_util.h>
 #include <grpc/support/time.h>

-#include "include/grpc/support/alloc.h"
+#include "grpc/support/alloc.h"
 #include "src/core/ext/filters/client_channel/client_channel.h"
 #include "src/core/ext/filters/client_channel/lb_policy.h"
 #include "src/core/ext/filters/client_channel/lb_policy/xds/xds.h"
